### DISCUSSION TITLE
Can particle birth be limited to a specific area, then allowed to move outside that area?

### DISCUSSION THREAD

**Discussion by AbeVigo** [matteobruni/tsparticles#1840] [Q&A] [answered]
Wondering if it is possible to limit particle birth to a specific area - for example, the bottom 25% of the canvas - then allow the birthed particles to move outside that area?

Ideally, not all particles would float up to the top, but a smaller portion. Possibly basing this on lifespan so all particles don't survive long enough to float up.

Here's an image to help illustrate the idea:

![image](https://user-images.githubusercontent.com/87538219/125974422-ff448c8c-76c3-4ef8-9545-38250dbfcbfe.png)

**Comment by matteobruni** [comment, marked as answer]
You can se a lot of samples of different sized emitters on my CodePen (https://codepen.io/matteobruni)

Here some samples:

https://codepen.io/matteobruni/pen/eYBaGxy
https://codepen.io/matteobruni/pen/NWRPKQW
https://codepen.io/matteobruni/pen/KKMrPON
https://codepen.io/matteobruni/pen/PoZgQea <-- this is a full screen emitter, particles will be created on all canvas area so you can see different emitters size options

**Reply by AbeVigo** [reply]
Thank you for the fast and helpful reply! The example at https://codepen.io/matteobruni/pen/NWRPKQW is right where I need to start.

Thanks again
